luisy/tasks/base.py

```diff
@@ -247,3 +247,9 @@ class DatabricksTask(Task):
     @property
     def spark(self):
         return Config().spark
+
+    def write(self, obj):
```

Comment:
should we call `obj` just `df` (because it must be one, either a spark df or a pandas df)?